### DISCUSSION TITLE
is there a way to download post that are minimum a day old?

### DISCUSSION THREAD

**Discussion by ashmonk** [UltimaHoarder/UltimaScraper#1424] [Q&A]
most of the promoposts have a timer of a day on them, so if i can downloqd  post that are at least a day old i dont need to wipe that stuf manually later.
or is there an other solution that i have missed?

thanks

**Comment by Nick05491** [comment]
I've also pondered a way to avoid downloading promo posts, and thought something like this may work.  However, there are some models who send content of themselves in messages that dissapears after a day, so this isn't optimal.  Best we could do may be to come up with words or phrases common to these promo posts (like "my bestie", "free today", etc.) and skip them.

The other idea I had would be to segregate these posts somehow, such as download them into a different 'For Review' folder or add a tag/comment to them within Finder (for Mac users).  Would at least make deleting the junk easier.
